src/components/input/input.html

```diff
@@ -0,0 +1,29 @@
+<div class="md-wrapper" (click)="input.focus()">
+  <div class="md-flex">
+    <prefix><ng-content select="[md-prefix]"></ng-content></prefix>
+
+    <label [attr.for]="'md-input-' + _inputId"
+           [class.md-empty]="empty" [class.md-focused]="focused"
+           [class.md-float]="floatingPlaceholder" [class.md-accent]="dividerColor == 'accent'">
+      {{placeholder}}
+      <span *ngIf="required">*</span>
+    </label>
+
+    <input #input aria-target
+           [class.md-end]="align == 'end'"
+           [attr.id]="'md-input-' + _inputId"
+           [disabled]="disabled" [required]="required"
+           [attr.maxlength]="maxLength" [attr.type]="type"
```

Comment:
`maxLength` is a real property on HTMLInputElement, so you don't need the `attr.`